workbench-core/swb-common-ui/package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "@aws/workbench-core-swb-common-ui",
+  "version": "0.1.2",
```

Comment:
```suggestion
  "version": "0.0.1",
```

Reply:
making this a minor change `0.1.0` and since library versions are affected in multiple packages (and they have to stay consistent), others will have a minor update as well. Let me know if you disagree @insignias.